src/core/quadtree.js

```diff
@@ -0,0 +1,462 @@
+/*jslint vars: true, nomen: true, plusplus: true, continue:true, forin:true */
+/*global Node, QNode */
+
+/*
+
+  Copyright (c) 2014 Michael Sander (speedplane)
+  Based loosely on code written by (c) 2011 Mike Chambers.
```

Comment:
@Snuffleupagus I originally started with a quadtree released under the MIT license. However, the original implementation had so many bugs and was changed so much that I feel pretty comfortable stating that it's a new work, not subject to the original license. Even if that isn't the case however, the MIT allows for derivative works, so we can just change the license over to MDL. I'll do that.